src/main/java/dk/cngroup/lentils/service/ProgressService.java

```diff
@@ -0,0 +1,47 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Progress;
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.repository.ProgressRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.temporal.TemporalAmount;
+import java.util.List;
+
+@Service
+public class ProgressService {
+
+    ProgressRepository progressRepository;
```

Comment:
Tohle bych označil jako private. Je to sice drobnost, ale pak to umožňuje měnit třídě již injektovanou závislost.
Pokud se tohle povoluje, pak se používá spíš Setter Injection přístup.